Tasks/MavenV3/maventask.ts

```diff
@@ -435,6 +435,7 @@ function publishCodeCoverage(isCodeCoverageOpted: boolean): Q.Promise<boolean> {
             }
             mvnReport.line(mavenOptions);
             mvnReport.arg("verify");
+            mvnReport.arg("-Dmaven.test.skip.exec");
```

Comment:
Could it make sense to use '-Dmaven.test.skip=true' instead? To avoid not only execution, but compiling also? According to http://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html

Reply:
Thank you! Replaced by this argument. Tested changes, works as expected